### ISSUE TITLE
accessibility issue 11, 31 (low) - fix heading levels

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#253]
Heading levels are skipped (h6 in the footer).  Headings should be organized and properly nested. Heading level should only increase by one.

There is more than one h1 heading level.  There should only be one h1>heading.  The h1 element should match the page title or have substantial meaning.

There should be a heading for search results.

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/9bf6c958ea633de9a5514f57b267b85c31dc165f

**Comment by yulgit1** [comment]
above also resolved issue 31:

Heading levels should not be skipped (h1 should be first on the page, then h2s, etc).  Headings should organize content (like an outline).  Better/more headings should be utilized to explain the information on the page (the “print made by…” could be a heading for the image and there could be a heading for the image details below that).  Visual users and users of assistive technology use headings to scan a page to understand what information is being presented.
